src/main/java/com/snowflake/kafka/connector/internal/SnowflakeErrors.java

```diff
@@ -192,6 +192,7 @@ public enum SnowflakeErrors {
   ERROR_3006("3006", "Failed to configure client status", "Exception reported by Ingest SDK"),
   ERROR_3007("3007", "Failed to get client status", "Exception reported by Ingest SDK"),
   ERROR_3008("3008", "Failed to ingest file with client info", "Exception reported by Ingest SDK"),
+  ERROR_3009("3009", "Streaming ingest client is null or closed", "The client must be initialized"),
```

Comment:
Maybe just `The client must be initialized`? Like what can the user do is they get a message saying the client is null.